variables.tf

```diff
@@ -367,13 +367,12 @@ variable "metadata_http_put_response_hop_limit" {
   default     = 2
   description = <<-EOT
     The desired HTTP PUT response hop limit (between 1 and 64) for Instance Metadata Service requests.
-    The default is `2` to support containerized workloads.
```

Comment:
Let's revert this line delete back unless you've got a reason for it. I see no reason to remove this additional bit of info.

Reply:
@Gowiem done